### PR TITLE
cull `base-python` and `base-python-test`

### PR DESCRIPTION
**Pull request by sherifnada**
## What
see #6164

remove `base-python`, `base-python-test` and all dependencies from the codebase. These are no longer used.

### CODE REVIEW

File: airbyte-cdk/python/setup.py [hunk 1]
```diff
@@ -70,7 +70,4 @@
             "sphinx-rtd-theme~=1.0",
         ],
     },
-    entry_points={
```
Comment (sherifnada) [inline airbyte-cdk/python/setup.py:73]: this is technically backwards breaking but i'm 99.999% sure only davin and I used this when we first wrote the CDK

File: airbyte-cdk/python/docs/tutorials/http_api_source.md [hunk 2]
```diff
@@ -312,7 +312,7 @@ Backoff policy options:
 - `max_retries` Specifies maximum amount of retries for backoff policy (by default is 5)
 - `raise_on_http_errors` If set to False, allows opting-out of raising HTTP code exception (by default is True)
 
-There are many other customizable options - you can find them in the [`base_python.cdk.streams.http.HttpStream`](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/base-python/base_python/cdk/streams/http.py) class. 
+There are many other customizable options - you can find them in the [`base_python.cdk.streams.http.HttpStream`](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/base_python/sources/streams/http.py) class.
```
Comment (sherifnada) [inline airbyte-cdk/python/docs/tutorials/http_api_source.md:315]:
```suggestion
There are many other customizable options - you can find them in the [`base_python.cdk.streams.http.HttpStream`](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py) class. 
```

**Review by pedroslopez — COMMENTED** [review]
I think this will break the build because of usage in `airbyte-source-test.gradle` here:

https://github.com/airbytehq/airbyte/blob/8983f09aea6826eecd2ea4443dad4c7aba69193a/buildSrc/src/main/groovy/airbyte-source-test.gradle#L33

will wait to approve upon removal, but looking good!

**Comment by sherifnada** [comment]
@pedroslopez removed that whole plugin since it's not used anywhere

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**